itou/www/signup/views.py

```diff
@@ -413,7 +441,8 @@ def prescriber_siret(request, template_name="signup/prescriber_siret.html"):
 
     session_data = request.session[settings.ITOU_SESSION_PRESCRIBER_SIGNUP_KEY]
 
-    form = forms.PrescriberSiretForm(data=request.POST or None, kind=session_data.get("kind"))
+    initial_data = {"siret": session_data["siren"]} if "siren" in session_data else None
```

Comment:
Ça marche aussi je pense :

```suggestion
    initial_data = {"siret": session_data.get("siren")}
```